Library/Homebrew/formula_installer.rb

```diff
@@ -236,6 +236,15 @@ def install
       raise CannotInstallFormulaError, message
     end
 
+    # Warn if a more recent version of this formula is available in the tap.
+    begin
+      if formula.pkg_version < (v = Formulary.factory(formula.full_name).pkg_version)
+        opoo "#{formula.name} #{v} is available and more recent than version #{formula.pkg_version}."
```

Comment:
Use `formula.full_name` here? @ilovezfs, does this look OK?

Reply:
Done.